src/func-cosmosdb-worker/src/functions/xxENSONOxx.xxSTACKSxx.Worker/CosmosDbChangeFeedEvent.cs

```diff
@@ -1,4 +1,4 @@
-using System.Text.Json.Serialization;
+using Newtonsoft.Json;
```

Comment:
Where was this used that it was able to be swapped out without other code changes?

Reply:
It was for the constructor attibute on line 7: -
*  [method: JsonConstructor]

I switched it for the Framework's version of the same name, but sicne realised the Service Bus library is using NewtonSoft, so thought I'd play it safe and be consistent.